tests/features/joinup_notification/solution.notification.feature

```diff
@@ -20,12 +20,17 @@ Feature: Solution notification system
       | Some Scent            | Some Scent            | logo.png | banner.jpg | The Red Search | Jody Buchanan       | deletion_request |
     And the following solution user memberships:
       | solution              | user             | roles |
-      | The Time of the Child | Benjamin Stevens | owner |
-      | Some Scent            | Benjamin Stevens | owner |
+      | The Time of the Child | Benjamin Stevens | owner, facilitator |
```

Comment:
I agree! This is a leftover from the change that Pieter introduced in a ticket and was missed for the solutions. It would be nice to change this here so that there are no issues later for our tests.